soh/soh/resource/type/AudioSample.h

```diff
@@ -20,6 +20,13 @@ namespace LUS {
         /* 0x08 */ s16* book; // size 8 * order * npredictors. 8-byte aligned
     } AdpcmBook; // s
 
+    enum class AudioFormat {
+        WAV = 0,
+        AIFF = 1,
+        MP3 = 2,
+        OGG = 3
+    };
```

Comment:
is the plan to use something other than `dr_libs` for aiff and ogg? i figure since we're using `dr_libs` this would make sense as

 ```suggestion
    enum class AudioFormat {
        WAV = 0,
        MP3 = 1,
        FLAC = 3
    };
```